src/core/api.client.ts

```diff
@@ -51,6 +53,7 @@ export class ApiClient {
     const defaultOptions: CycleTLSRequestOptions = {
       ja3: '771,4865-4867-4866-49195-49199-52393-52392-49196-49200-49162-49161-49171-49172-51-57-47-53-10,0-23-65281-10-11-35-16-5-51-43-13-45-28-21,29-23-24-25-256-257,0',
       userAgent: 'KICK/1.0.13 Dalvik/2.1.0(Linux; U; Android 13; Pixel 6 Pro Build / TQ1A.221205.011)',
+      ...(this.proxy && {proxy: this.proxy}),
       headers: {
```

Comment:
Nitpick, keep linter happy

```suggestion
      ...(this.proxy && { proxy: this.proxy }),
```